src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -133,31 +142,29 @@ public static ToaMistake toGroupedMistake(ToaMistake mistake) {
      *                     previously before this.
      * @return The mistake chat message to use for the raider
      */
-    public static String getChatMessageForMistakeCount(ToaMistake mistake, int mistakeCount) {
+    public static String getChatMessageForMistakeCount(ToaMistakeTrackerConfig config, ToaMistake mistake, int mistakeCount) {
         // Special case a few mistake chat messages
-        if (mistake == AKKHA_UNSTABLE_ORB) {
-            return getChatMessageForAkkhaUnstableOrb(mistake.getChatMessage(), mistakeCount);
-        } else if (mistake == BABA_SLAM) {
-            return getChatMessageForBabaSlam(mistake.getChatMessage(), mistakeCount);
+        if (config.stackQuestionMarks() && mistake.getChatMessage(config).equals("?")) {
+            return getStackingChatMessage(mistake.getChatMessage(config), mistakeCount);
+        } else if (mistake.getChatMessage(config).contains("|")) {
+            return getAlternatingChatMessage(mistake.getChatMessage(config), mistakeCount);
         }
-
-        return mistake.getChatMessage();
+        
+        return mistake.getChatMessage(config);
```

Comment:
nit: Save `mistake.getChatMessage(config)` to a variable at the top of the method so it can be reused throughout, instead of re-computing it